### ISSUE TITLE
Improve placement of Notices which prevents users from editing Analytics settings

### ISSUE THREAD

**Issue by wpdarren** [google/site-kit-wp#6224]
## Bug Description

While testing #5886 I noticed some inconsistencies with the notices when a second admin tries to make changes to Analytics when they do not have access.  As you can see from the screenshot, for UA property, the notice is below the dropdown fields where for GA4 the notice appears further down. To be consistent we should have the notice in the same location for UA and GA4.

![image](https://user-images.githubusercontent.com/73545194/204493948-a0febfca-57f3-4503-8374-013710c2fe01.png)

As a separate but linked minor issue the icon for the notice looks quite large compared with the text and there's too much padding around it and the message. Not sure if we have any direction with UX here but looks odd. See screenshot above.

## Steps to reproduce

See the steps in #5886 to recreate this issue.  Edit the Analytics settings as Admin2 and see the placement of the notices. 

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* The notice shown in the Analytics settings edit view for non-owner admins should be placed consistently both for UA and GA4, immediately below the primary connection controls (i.e. account, property), but above secondary (e.g. snippet toggle)

## Implementation Brief

* Inside `google-site-kit/assets/js/modules/analytics/components/settings/GA4SettingsControls.js`
    * Locate `<GA4SettingsNotice />` and move it to render above `<GA4ActivateSwitch />`

### Test Coverage

* VRT images might need to be updated

## QA Brief

* Try to make changes to Analytics when do not have access
* check notices

## Changelog entry

* Apply consistent placement of notices which inform users they don't have permission to edit Analytics settings.

**Comment by tofumatt** [comment]
IB ✅

**Comment by wpdarren** [comment]
### QA Update: ⚠️ 

@techanvil I know you did not complete this initial work but in Sasha's absence.  This is possibly out of scope for this ticket and maybe I've found a bug that is just linked to the 2nd admin user.  

When I login as a second admin who does not have access to the Analytics account/property, there are a few observations.

The site did not have the GA4Reporting flag enabled and has a UA and GA4 setup for it.  It's currently in gathering state. 

All looks good with admin 1. 

For when I login as admin 2 who does not have access.  In Edit Analytics settings:

1. The GA4 account and property loading spinner appears and does not complete.  

2. The messages do appear in the correct place, but what I see now is a similar permissions error in red font that wasn't appearing previously. 

![image](https://user-images.githubusercontent.com/73545194/235933697-08968383-13ce-465a-933f-cd866830b96b.png)

3. When I login as admin 2 with `GA4Reporting` feature flag enabled, I see a similar situation. 

What do you think? 

![image](https://user-images.githubusercontent.com/73545194/235936105-0a4b0a8c-187a-4d0f-9438-27d1e004188f.png)

**Comment by techanvil** [comment]
Hi @wpdarren, thanks for raising this. It is indeed outside of the scope of this issue, which is purely about moving the position of the GA4 notice to be directly below the dropdowns.

I think the new errors in red are appearing as a result of the work in https://github.com/google/site-kit-wp/issues/6831. It doesn't seem problematic to me, although we could discuss it on a separate issue to see if we'd like to prevent certain types of error messages from displaying when we're also showing the grey notices.

The dropdowns being stuck in a loading state does look like something we need to fix, I don't see there being a question about that. I'd suggest creating a separate issue for this so we can fix it without being held up by discussion about the red errors.

**Comment by aaemnnosttv** [comment]
LGTM 👍
